src/test/java/com/hubspot/jinjava/tree/ExpressionNodeTest.java

```diff
@@ -30,7 +31,19 @@ public void itRendersResultAsTemplateWhenContainingVarBlocks() throws Exception
     context.put("place", "world");
 
     ExpressionNode node = fixture("simplevar");
-    assertThat(node.render(interpreter).toString()).isEqualTo("hello {{ place }}");
+    assertThat(node.render(interpreter).toString()).isEqualTo("hello world");
+  }
+
+  @Test
+  public void itRendersResultWithoutInterpretateExpression() throws Exception {
```

Comment:
I think there's still a typo in there. Should be `itRendersResultWithoutInterpreterExpression`